compiler/semstmts.nim

```diff
@@ -241,6 +246,10 @@ proc semTry(c: PContext, n: PNode; flags: TExprFlags): PNode =
     elif a.kind != nkFinally:
       illFormedAst(n, c.config)
 
+    if n_catch_all_excepts > 1:
+      # if number of ``except: body`` blocks is greater than 1, it is invalid
+      illFormedAst(n, c.config)
```

Comment:
I've read you wanted to show a meaningful error message here, have a look at `localError` and how it's used.